Formula/gnat.rb

```diff
@@ -0,0 +1,157 @@
+class Gnat < Formula
+  desc "GNU New York University Ada Trans"
+  homepage "https://gcc.gnu.org/"
+  url "https://ftp.gnu.org/gnu/gcc/gcc-11.1.0/gcc-11.1.0.tar.xz"
+  mirror "https://ftpmirror.gnu.org/gcc/gcc-11.1.0/gcc-11.1.0.tar.xz"
+  sha256 "4c4a6fb8a8396059241c2e674b85b351c26a5d678274007f076957afa1cc9ddf"
+  license "GPL-3.0-or-later" => { with: "GCC-exception-3.1" }
+  head "https://gcc.gnu.org/git/gcc.git"
+
+  livecheck do
+    url "https://ftp.gnu.org/gnu/gcc/gcc-11.1.0"
+    regex(%r{href=.*?gcc[._-]v?(\d+(?:\.\d+)+)(?:/?["' >]|\.t)}i)
+  end
+
+  # The bottles are built on systems with the CLT installed, and do not work
+  # out of the box on Xcode-only systems due to an incorrect sysroot.
+  pour_bottle? do
+    on_macos do
+      reason "The bottle needs the Xcode CLT to be installed."
+      satisfy { MacOS::CLT.installed? }
+    end
+  end
```

Comment:
@Rylan12 we have new syntax for this now right?

Reply:
Yes but we need to wait for a new tag for it to be used in Homebrew/core.

Once that's been made, it should be:

```suggestion
pour_bottle? only_if: :clt_installed
```

If desired this can be merged and I'll update it to the new syntax when I do the rest of the formulae.